packages/client/e2e/pages/FxNewRfqPageObject.ts

```diff
@@ -0,0 +1,31 @@
+import { Page, TestInfo } from "@playwright/test"
+
+export class FxNewRfqPageObject {
+  constructor(
+    readonly page: Page,
+    readonly testInfo: TestInfo,
+  ) {}
+
+  async clickCurrency(currency: string) {
+    return await this.page
+      .getByTestId("tab-bar-tabs")
+      .getByText(currency)
+      .click()
+  }
+
+  get spotTileNotionalInput() {
+    return this.page.locator("input[id='notional-input-EURUSD']")
+  }
+
+  async buyCurrencyPair(pair: string) {
+    return await this.page.getByTestId(`Buy-${pair}`).click()
+  }
+
+  async sellCurrencyPair(pair: string) {
+    return await this.page.getByTestId(`Sell-${pair}`).click()
+  }
+
+  get tradeId() {
```

Comment:
I am leaning towards the pattern of having all the getters at the top .. happy to discuss this pattern, tho